pyhstr/application.py

```diff
@@ -18,9 +20,15 @@ def __init__(self, stdscr):
         self.stdscr = stdscr
         self.user_interface = UserInterface(self)
         self.all_entries = {
-            0: sort(read(PYTHON_HISTORY)),
-            1: sort(read(FAVORITES)),
-            2: remove_duplicates(read(PYTHON_HISTORY))
+            0: sort(
+                read(PYTHON_HISTORY) if SHELL == 'regular' else
+                get_ipython_history() 
+            ),
+            1: sort(read(f'{FAVORITES}-{SHELL}')),
+            2: remove_duplicates(
+                read(PYTHON_HISTORY) if SHELL == 'regular' else
+                read_ipython_history() 
```

Comment:
```suggestion
                get_ipython_history()
```

I don't like the copy/pasta with two reads. How about something like this:

```
history = read(PYTHON_HISTORY) if bla else bla
self.all_entries = {
    0: sort(history),
    1: ...,
    2: remove_duplicates(history),
}
```